WPMedia/fusion-news-theme-blocks
# Tmedia 468 header nav chain buttons

## Description
Header nav chain buttons, querly, arc-search, sections

## Jira Ticket
- [TMEDIA-468](https://arcpublishing.atlassian.net/browse/TMEDIA-468)

## Acceptance Criteria

- Match color logic for nav color and borders for sections, queryly, arc search (not expanded)
- Expanded input should always have white background and incorporate the button similarly as before

## Test Steps

1. Checkout this branch `git checkout TMEDIA-468-header-nav-chain-buttons`
2. Run fusion repo with linked blocks `npx fusion start -f -l  @wpmedia/identity-block,@wpmedia/shared-styles,@wpmedia/header-nav-chain-block`
3. Create homepage with sections, queryly on the left and arc search on the right area
4. Show the page on all three potential nav color iterations the-sun (primary-color), dagen (light), and the-prophet (dark) 
5. Should match color effects based on mocks 

<img width="456" alt="Screen Shot 2021-09-27 at 15 01 19" src="https://user-images.githubusercontent.com/5950956/134976893-6a162abe-c6e8-4395-9a72-75a5c64b8108.png">

## Effect Of Changes
### Before
- Dark and light color only 
- Used inconsistent border color 

### After
<img width="1372" alt="Screen Shot 2021-09-27 at 15 03 30" src="https://user-images.githubusercontent.com/5950956/134977231-d0b03c6c-34a2-43d6-8164-bb13e80246bf.png">


## Dependencies or Side Effects
- Shared styles scss has been changed

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored). -> removed theme color params to use properties

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
